### PR TITLE
Listen Later Feature

### PR DESCRIPTION
**Pull request by shrey1608**

### CODE REVIEW

**Comment by shrey1608** [comment]
Please review it @ExpressHermes

File: fpage/views.py [hunk 1]
```diff
@@ -21,6 +22,34 @@
 #class DetailView(generic.DetailView):
   #  model = Album
    # template_name='fpage/detail.html'
+
+def watchlater(request):
+    if request.method == "POST":
```
Comment (ExpressHermes) [inline fpage/views.py:27]: Write comments to explain the logic of this view. You can mention only the parts that will be hard to understand.

File: fpage/templates/fpage/watchlater.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{% extends 'fpage/base.html' %}
+{% block content %}
+{% load static %} 
+<h1 style="color: rgb(158, 60, 60); text-align: center;">Watch Later</h1>
```
Comment (ExpressHermes) [inline fpage/templates/fpage/watchlater.html]: Put this is style.css

**Comment by shrey1608** [comment]
Any updates @ExpressHermes . I have a PR waiting in line

**Comment by ExpressHermes** [comment]
@shrey1608 I have added comments about issue that need to be addressed. Resolve them. Ping me when you are done. I will merge it

**Comment by shrey1608** [comment]
Umm @ExpressHermes  i cant see any comments where are they mentioned? Please help me out

File: fpage/views.py (hunk 1 above)
Comment (ExpressHermes) [reply fpage/views.py:27]: @shrey1608

File: fpage/templates/fpage/watchlater.html (hunk 2 above)
Comment (ExpressHermes) [reply fpage/templates/fpage/watchlater.html]: @shrey1608

**Comment by shrey1608** [comment]
The changes suggested by the djnago doctor? @ExpressHermes

**Comment by ExpressHermes** [comment]
No, I have commented on the codes that needs to be changed. 

- In the watch_later view, write comments as some of the logic will be hard for anyone to figure out without comments. 

- in the new html files, you have added CSS using style tag, delete the tags and place the cas in static file

**Comment by shrey1608** [comment]
@ExpressHermes made the changed

**Comment by shrey1608** [comment]
Any updates @ExpressHermes

**Review by ExpressHermes — APPROVED** [review]

**Comment by shrey1608** [comment]
please provide points @ExpressHermes . I think this will be a 150 point as i implemented the listen later feature and also its landing page and logic was tough to implement as i also made the listen later sort based on time and not on song id  😅

**Comment by shrey1608** [comment]
@ExpressHermes please provide points here

**Comment by ExpressHermes** [comment]
@mann27 decide appropriate points here.

**Comment by shrey1608** [comment]
if @mann27 doesnt come online please give points as 12 pm is the deadline for giving points and contribution @ExpressHermes Thanks!

**Comment by ExpressHermes** [comment]
@shrey1608 In my opinion this issue is worth 100 points.

 This is just a suggestion: The sorting problem could have been tackled much better if you have added a `DateTimeField` in watch_later model. Then you could have used order_by while filtering.

**Comment by shrey1608** [comment]
okay i didnt think of this 😅. As you guys say would be final for me. Can you give like 100 now and if mann feels its worth 150 he can add more :) @ExpressHermes

**Comment by ExpressHermes** [comment]
Okay

**Comment by ExpressHermes** [comment]
give @shrey1608 75 points

**Comment by ExpressHermes** [comment]
give @shrey1608 25 points

**Comment by shrey1608** [comment]
ok Thanks! Contest is over right? @ExpressHermes @mann27

**Comment by mann27** [comment]
probably can't say fs. Sorry I wasn't available for all the mentions. I think what @ExpressHermes has given as points are the final points as he was the one going through the code so I will not interfere with his decisions. 

Thanks for all the contributions @shrey1608! 😀

**Comment by shrey1608** [comment]
okay like the mail mentioned 23rd november 12 Noon is the deadline so i think it would have ended. Anyways Thanks a lot @ExpressHermes @mann27 for guiding me and i have learnt a lot through out the time period and i wish to stay connected to both of you and learn more. I will be making contribution to albums as some things are yet to be implemented. Thanks!

**Comment by ExpressHermes** [comment]
It was fun guiding you and reviewing all the contributions.

**Pull request merged**